implementations/rust/c/bindings/build.rs

```diff
@@ -40,9 +56,35 @@ fn main() {
     );
     println!("cargo:rustc-link-lib=static=ockam_vault_atecc608a");
 
+    println!(
```

Comment:
Is `log` a package? If not, I'm confused as to why we would link to a log directory?